.github/workflows/pr-metadata.yml

```diff
@@ -16,15 +16,15 @@ jobs:
     name: Report Jira data to PR comment
     steps:
       - name: Checkout
-        uses: actions/checkout@v3
+        uses: actions/checkout@v4
```

Comment:
Just for fun this action triggers on pull_request_target so that it works for PRs across forks but that does mean that this PR is running the code from master rather than the code from the PR and this action does interact with git histories as well.